_summaries/misra2019mish_2.md

```diff
@@ -1,19 +1,34 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: misra2019mish_2
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: biofizzatreya
+score: 7
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+This paper attempts to fix the vanishing gradient problem of Relu activation functions by using, what they call Mish. Mish is defined as $$ x\cdot tanh\left(\text{ln}(1+e^x)\right) $$. The authors compare the effect of using different activation functions such as Relu, Swish and Mish. 
+
+![Picture1](https://user-images.githubusercontent.com/13065170/132050780-f3438c1c-a2b6-433d-a3de-2d9279d81cce.png)
```

Comment:
Unsure comment: Not sure if it's okay to include the explanation sections for the tables since in the summary guide it says "use your own words (do not copy from the paper)". Maybe it would be better to explain those sections on your own, its your call.